src/content/docs/design/content-customization/how-content-is-managed.mdx

```diff
@@ -0,0 +1,28 @@
+---
+page_id: b3a309c6-af3c-4155-b39e-7e1f6e83721f
+title: How content is managed in Kinde
+sidebar:
+  order: 1
+relatedArticles:
+  - f0bc688b-a817-42ab-9a20-8e09cec06f37
+
+---
+
+You can change the content for the main pages that your users see as part of authentication. You’re also able to manage the content in [all the languages you have selected](/design/pages/internationalization/) in Kinde.
+
+Currently, you can edit content for the following pages:
+
+- Sign up
+- Sign-up confirmation
+- Sign in
+- Sign-in confirmation
+- Request access form
+
+Additional screens will be added, including error screens, in the coming months.
+
+## What can be edited?
+
+All the content elements on pages, including headings, descriptions, field labels, etc. can be edited.
```

Comment:
@clairekinde11 as of now, it's still just the handful of fields we allowed previously we haven't opened the floodgates yet.

We added edit functionality for these few fields that had high customer asks:

Sign up screen
* email label

Sign in screen
* email label
* Account not found error message